templates/cf.yml

```diff
@@ -631,6 +654,9 @@ properties:
     blacklisted_syslog_ranges: ~
     unmarshaller_count: (( merge || 5 ))
     port: (( merge || 4443 ))
+    etcd:
+      client_cert: ~
+      client_key: ~
```

Comment:
I see.  So we'll need to generate a lot of different etcd client certs.  There's a question of what should be possible in the manifest, and what should be possible via the spiff templates.  We should be able to configure the client certs independently in the manifest.  Whether the spiff workflow should do that is a separate question.  The spiff templates have a confusing use cases, they're supposed to be both easy to get started with, but some people also want it to be hardened for production deployments too.  We err on the side of simplicity, because there's no way to guarantee spiff templates are production hardened.  Let's discuss whether it makes sense, for the purpose of these templates, to just copy around the same client certs.